packages/cli/lib/projectStructure.js

```diff
@@ -84,24 +97,21 @@ async function findProjectComponents(projectSourceDir) {
 
   projectFiles.forEach(projectFile => {
     // Find app components
-    if (projectFile.endsWith(APP_COMPONENT_CONFIG)) {
+    const { base, dir } = path.parse(projectFile);
+
+    if (Object.values(CONFIG_FILES).includes(base)) {
       const parsedAppConfig = loadConfigFile(projectFile);
 
       if (parsedAppConfig && parsedAppConfig.name) {
-        const appPath = projectFile.substring(
-          0,
-          projectFile.indexOf(APP_COMPONENT_CONFIG)
-        );
-        if (typeof appPath === 'string') {
-          const isLegacy = getIsLegacyApp(parsedAppConfig, appPath);
-
-          components.push({
-            type: COMPONENT_TYPES.app,
-            config: parsedAppConfig,
-            runnable: !isLegacy,
-            path: appPath,
-          });
-        }
+        const appPath = `${dir}/`;
```

Comment:
You may need to use `path` here instead of manually inserting the slash to make sure it's compatible with other OS's

Reply:
Oooh good call

Reply:
Just tested and looks like it doesn't actually need the slash